packages/sdk/src/protocols/filter/constants.ts

```diff
@@ -1,4 +1,4 @@
-export const DEFAULT_KEEP_ALIVE = 30 * 1000;
```

Comment:
Ideally we should be fine with increasing this number, not decreasing.
How did you reach this number?